Mission 3/NMCA_003/NMCA_003_strings.lua

```diff
@@ -1,9 +1,455 @@
+}
+
+-- Spawn ACU and move cam to it
+M1Intro3 = {
+    {text = '[Nichols]: There\'s the ship. We have no time to waste, sending you in commander.', vid = 'N03_M1Intro3.sfd', bank = 'N03_VO', cue = 'M1Intro3', faction = 'UEF'},
+}
+
+
+
+-- Assign primary objectives, protect ship, kill aeon base
+M1PostIntro = {
+    {text = '[Benson]: Commander, Aeon units are closing to your position, secure the area and defend the ship.', vid = 'N03_M1PostIntro_1.sfd', bank = 'N03_VO', cue = 'M1PostIntro_1', faction = 'UEF'},
+    {text = '[Benson]: Once you establish a base, send your units to the south and destroy the Aeon base.', vid = 'N03_M1PostIntro_2.sfd', bank = 'N03_VO', cue = 'M1PostIntro_2', faction = 'UEF'},
+}
+
+-- Player kills the first Aeon base
+M1AeonBaseDestroyed = {
+    {text = '[Nichols]: The Aeon base has been destroyed, good job.', vid = 'N03_M1AeonBaseDestroyed.sfd', bank = 'N03_VO', cue = 'M1AeonBaseDestroyed', faction = 'UEF'},
+}
+
+-- Reminder #1 to kill Aeon base
+M1AeonBaseReminder1 = {
+    {text = '[Benson]: The Aeon base is still a threat, destroy it.', vid = 'N03_M1AeonBaseReminder1.sfd', bank = 'N03_VO', cue = 'M1AeonBaseReminder1', faction = 'UEF'},
+}
+
+-- Reminder #2 to kill Aeon base
+M1AeonBaseReminder2 = {
+    {text = '[Benson]: The base wont kill itself commander.', vid = 'N03_M1AeonBaseReminder2.sfd', bank = 'N03_VO', cue = 'M1AeonBaseReminder2', faction = 'UEF'},
+}
+
+
+
+-- When the ship gets damaged
+M1ShipDamaged = {
+    {text = '[Benson]: Commander, the ship is under attack! Protect it!', vid = 'N03_M1ShipDamaged.sfd', bank = 'N03_VO', cue = 'M1ShipDamaged', faction = 'UEF'},
+}
+
+-- When the ship drops to half HP
+M1ShipHalfDead = {
+    {text = '[Benson]: The ship is taking damage!', vid = 'N03_M1ShipHalfDead.sfd', bank = 'N03_VO', cue = 'M1ShipHalfDead', faction = 'UEF'},
+}
+
+-- When the ship is nearly destroyed
+M1ShipAlmostDead = {
+    {text = '[Benson]: The ship is almost dead!', vid = 'N03_M1ShipAlmostDead.sfd', bank = 'N03_VO', cue = 'M1ShipAlmostDead', faction = 'UEF'},
+}
+
+
+
+-- First Aeon units close the the player
+M1AeonAttackWarning = {
+    {text = '[Benson]: Aeon units are closing it to your position, get ready to fight.', vid = 'N03_M1AeonAttackWarning.sfd', bank = 'N03_VO', cue = 'M1AeonAttackWarning', faction = 'UEF'},
+}
+
+
+
+-- Player sees the dead UEF Base
+M1UEFBaseDialogue = {
+    {text = '[Anyone]: Something about the dead base.', vid = 'N03_M1UEFBaseDialogue.sfd', bank = 'N03_VO', cue = 'M1UEFBaseDialogue', faction = 'UEF'},
+}
+
+
+
+-- Once player scouts the civilian city, add objective to capture one building
+M1SecondaryObjective = {
+    {text = '[Benson]: We\'ve located a small civilian settlement south of your position. Capture the marked building to get an additional intel on this area.', vid = 'N03_M1SecondaryObjective.sfd', bank = 'N03_VO', cue = 'M1SecondaryObjective', faction = 'UEF'},
```

Comment:
instead of 
Capture the marked building to gain additional intel on this area.
something like "we might find more information there..." cause its an unknown enemy

Some more explanation why to capture it